client/src/components/MainLayout/MainLayout.js

```diff
@@ -3,6 +3,7 @@ import { Outlet } from "react-router-dom";
 
 import NavBar from "../NavBar/NavBar";
 import Footer from "../Footer/Footer";
+import ResponsiveAppBar from "../ResponsiveAppBar/ResponsiveAppBar";
```

Comment:
In this file, you imported the new ResponsiveAppBar. The main component in this file is still using `Navbar`. 
- In line 15 of this file, you will be replacing `NavBar` with `ResponsiveAppBar`
- then delete the import of `NavBar` in line 4.

To run the app using npm start first: 
- open the integrated terminal in VsCode
- run `cd client` on the terminal to change into the client folder
- run `npm start` to start the app and launch it on browser
You should now see the new app navbar in the browser. Let me know if you have any issues

Reply:
Great, just did that. However, **npm start** gave me the following output:

            > ecohabit@0.1.0 start
            > react-scripts start
            sh: react-scripts: command not found

I looked it up on [stack overflow](https://stackoverflow.com/questions/47612580/react-scripts-command-not-found) and found out there might be issues with packages installation. What would be your recommendation?

Reply:
@mahir118, do you see a folder named "node_modules" in the "client" folder? if not, please use `npm install` and wait for the installation complete. after that try `npm start` again.

Reply:
Got it, it's working perfectly now. Thanks a lot for all your help. Is there anything else I can do from this point?